apps/web/.env

```diff
@@ -87,7 +87,13 @@ NEXT_PUBLIC_DISABLE_AUTO_REFRESH=false
 APP_NAME="Ever Teams"
 APP_SIGNATURE="Ever Teams"
 APP_LOGO_URL="https://app.ever.team/assets/ever-teams.png"
+NEXT_PUBLIC_APP_NAME="Ever Teams"
+NEXT_PUBLIC_COMPANY_NAME='Ever Co. LTD'
+NEXT_PUBLIC_LOGO_URL=
```

Comment:
@samuelmbabhazi why you added NEXT_PUBLIC_LOGO_URL if we already had APP_LOGO_URL?
Same for NEXT_PUBLIC_APP_NAME while we had APP_NAME and so on!?